src/python/install.sh

```diff
@@ -26,7 +26,7 @@ CONFIGURE_JUPYTERLAB_ALLOW_ORIGIN="${CONFIGUREJUPYTERLABALLOWORIGIN:-""}"
 # alongside PYTHON_VERSION, but not set as default.
 ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
-DEFAULT_UTILS=("pylint" "flake8" "autopep8" "black" "yapf" "mypy" "pydocstyle" "pycodestyle" "bandit" "pipenv" "virtualenv" "pytest")
+DEFAULT_UTILS=("pylint" "flake8" "autopep8" "black" "yapf" "mypy" "pydocstyle" "pycodestyle" "bandit" "pipenv" "virtualenv" "pytest" "ruff")
```

Comment:
Can we bump the minor version as we are adding this new util? thanks! 

https://github.com/devcontainers/features/blob/4c2ba73417ed66d90b3198e906bb89edc4f2f020/src/python/devcontainer-feature.json#L3

Reply:
Can you help add similar tests/checks for `ruff`?

https://github.com/search?q=repo%3Adevcontainers%2Ffeatures%20check%20%22autopep8%22%20autopep8%20--version&type=code

Reply:
comment on lines

Reply:
My apologies for the delay. I have both added the `check` tests and bumped the minor version.